result_addon.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type addonCopy struct {
+	sourceTestOutputDir   string
+	targetAddonPath       string
+	targetAddonBundleName string
+}
+
+func copyToResultAddonDir(info addonCopy) error {
```

Comment:
could be 2 functions, just like here: https://github.com/bitrise-io/bitrise/pull/647/files/1486ee52ac69d237d775bc366f1677555cee67a6#r252625893

may the first copy function could return the addonPerStepOutputDir and the second could use the dir to save the testBundle